src/pages/friend.js

```diff
@@ -39,166 +39,154 @@ function Friend() {
-                  }
-                  onMouseLeave={(e) => (e.target.innerText = "Battleship")}
-                >
-                  Battleship
-                </button>
-              </Link>
-
-              <Link to="https://toughlovearena.com/">
-                <button
-                  class="w-72 h-20 relative flex flex-col items-center justify-center w-72 h-20 rounded-[27px] text-white text-lg font-bold bg-cover bg-center focus:border-4 focus:border-white"
-                  style={{ backgroundImage: `url(${OnlineMiniGames})` }}
-                  onMouseOver={(e) =>
-                    (e.target.innerText =
-                      "Play a fun mini-game with your friend!")
-                  }
-                  onMouseLeave={(e) =>
-                    (e.target.innerText = "Online mini-games")
-                  }
-                >
-                  Online mini-games
-                </button>
-              </Link>
-
-              <Link to="https://www.chess.com">
-                <button
-                  class="w-72 h-20 relative flex flex-col items-center justify-center w-72 h-20 rounded-[27px] text-white text-lg font-bold bg-cover bg-center focus:border-4 focus:border-white"
-                  style={{ backgroundImage: `url(${Chess})` }}
-                  onMouseOver={(e) =>
-                    (e.target.innerText =
-                      "Play the classic chess game with your friend")
-                  }
-                  onMouseLeave={(e) => (e.target.innerText = "Chess")}
-                >
-                  Chess
-                </button>
-              </Link>
-
-              <Link to="https://www.trivianerd.com/">
-                <button
-                  class="w-72 h-20 relative flex flex-col items-center justify-center w-72 h-20 rounded-[27px] text-white text-lg font-bold bg-cover bg-center focus:border-4 focus:border-white"
-                  style={{ backgroundImage: `url(${TriviaGames})` }}
-                  onMouseOver={(e) =>
-                    (e.target.innerText = "Answer questions about any topic!")
-                  }
-                  onMouseLeave={(e) => (e.target.innerText = "Trivia Games")}
-                >
-                  Trivia Games
-                </button>
-              </Link>
-            </div>
+          <div className="grid gap-16 justify-center mt-12">
+            <Link to="https://skribbl.io">
+              <button
+                className="w-72 h-20 relative flex flex-col items-center justify-center w-72 h-20 rounded-[27px] text-white text-lg font-bold bg-cover bg-center focus:border-4 focus:border-white"
+                style={{ backgroundImage: `url(${Skriblio})` }}
+                onMouseOver={(e) =>
+                  (e.target.innerText =
+                    "Draw and guess what your friends are drawing")
+                }
+                onMouseLeave={(e) => (e.target.innerText = "Skribl.io")}
+              >
```

Comment:
Mixing logic handling on markup is not a good practice. Is not completely wrong tho.
The correct approach would be creating a function on the component ex `mouseLeaveHandler` and referring to it on the markup. That way, markup will just be markup, and logic will be contained